### PR TITLE
Arman: tasks 61 70

### PR DESCRIPTION
**Pull request by hsp-arman**
Tasks 61-70

### CODE REVIEW

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_63.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * Print "1" (answer1) , if any digit of number is equal to COMPARABLE_VALUE , else print "0" (answer2)
+ * 
+ * @author arman
+ *
+ */
+public class Task_63 extends TaskDigit {
+
+	private static final int COMPARABLE_VALUE = 1;
+	private static int answer1 = 1;
```
Comment (hsp-hripsime) [inline src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_63.java]: Constants must be final and all letters in their names must be uppercase.

File: src/main/java/am/hsp/apps/seua/base/TaskDigit.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	 * @param comparableNumber
+	 * @return
+	 */
+	protected boolean isEqualTo(int number, int comparableNumber) {
+		for (int i = 1; i <= getCountOfDigit(number); i++) {
+			if (getDigit(number, i) == comparableNumber) {
+				return true ;
+			}
+		}
+		return false;
```
Comment (hsp-hripsime) [inline src/main/java/am/hsp/apps/seua/base/TaskDigit.java]: This method must be also changed, as contains 2 cycles. Besides this, the method name isn't descriptive.

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_64.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ * Print char 's' (answer1) , if sum of ONES and TENS is equal to 5
+ * (COMPARABLE_VALUE) , else return char 'd' (answer2)
+ * 
+ * @author arman
+ *
+ */
+public class Task_64 extends TaskDigit {
+	
+	private static final int COMPARABLE_VALUE = 5 ;
+	private static char answer1 = 's';
```
Comment (hsp-hripsime) [inline src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_64.java]: The same problem is here.

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_67.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	/**
+	 * return String "YES" (answer1) , if number is equal to square sum of digits ,
+	 * else return String "NO" (answer2)
+	 * 
+	 * @param number
+	 * @return
+	 */
+	private String getResult(int number) {
+		int sum = getSumOfDigits(number);
+		return number == sum * sum ? answer1 : answer2;
```
Comment (hsp-hripsime) [inline src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_67.java]: You can use here getPow() method.

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_68.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ * Print product ONES and HUNDREDS , if ONES > TENS , else "1" (CONSTANT)
+ * 
+ * @author arman
+ *
+ */
+public class Task_68 extends TaskDigit {
+
+	private static final int CONSTANT = 1;
```
Comment (hsp-hripsime) [inline src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_68.java]: I don't know is it a preferable name?

Comment (hsp-harut) [reply src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_68.java]: :+1: of course no. Arman jan never use such names  CONSTANTS MUST HAVE DESCRIPTIVE NAMES. This is the same if you will name an "int" variable as "integer"... please fix

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_67.java (hunk 4 above)
Comment (hsp-harut) [reply src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_67.java]: :+1: absolutely agree

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_63.java (hunk 1 above)
Comment (hsp-harut) [reply src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_63.java]: :+1: very good point

File: src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_64.java (hunk 3 above)
Comment (hsp-harut) [reply src/main/java/am/hsp/apps/seua/Tasks_61_70/Task_64.java]: :+1:

File: src/main/java/am/hsp/apps/seua/tasks_21_30/Task_30.java [hunk 6]
```diff
@@ -52,7 +52,7 @@ private static int getSum(int number1, int number2, int number3) {
 	 * @param number2
 	 * @param number3
 	 */
-	private int getMid(int number1, int number2, int number3) {
+	protected int getMid(int number1, int number2, int number3) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_21_30/Task_30.java]: why you changed the access modifier to protected?

**Pull request merged**